src/wp-includes/class-wp-hook.php

```diff
@@ -80,6 +80,11 @@ final class WP_Hook implements Iterator, ArrayAccess {
 	 * @param int      $accepted_args The number of arguments the function accepts.
 	 */
 	public function add_filter( $hook_name, $callback, $priority, $accepted_args ) {
+		// force the priority to be numeric, if it wasn't, set to the default priority
+		if ( ! is_numeric( $priority ) ) {
+			$priority = 10;
+		}
+		
```

Comment:
```suggestion
		// Ensure the priority is be numeric, if it wasn't, set to the default priority.
		if ( ! is_numeric( $priority ) ) {
			$priority = 10;
		}

```

Fixed the sniffer message and the comment to match WPCS.

Reply:
Addtional to my comment, I suggest to add to add a `_doing_it_wrong()` here to ensure the developer finds this.

```suggestion
		// Ensure the priority is be numeric, if it wasn't, set to the default priority.
		if ( ! is_numeric( $priority ) ) {
			_doing_it_wrong( 
				'add_filter',
				sprintf(
					/* translators: 1: 'add_action()'. 2: 'add_filter()'. */
					__( 'Priority for %1$s and %2$s needs to a numeric value.' ),
					'<code>add_action()</code>',
					'<code>add_filter()</code>'
				),
				'6.8'
			); 
			$priority = 10;
		}

```